src/components/non-blocking-forms/NonBlockingErrors.vue

```diff
@@ -11,6 +11,7 @@
 import { useValidationMessages } from "./useValidationMessages";
 
 const props = defineProps<{
+  showMessages?: boolean;
```

Comment:
This new prop is never filled in any of the implementations it seems. 

Therefore, it seems redundant.

Reply:
Fair enough. I was a bit lazy with removing the template code. I cleaned it up now.